src/main/java/base/BaseTest.java

```diff
@@ -19,9 +19,10 @@ public BaseTest() {
         PageFactory.initElements(driver, this);
         action = new Actions(driver);
     }
-
-    private WebDriverWait waitPage(){
+    public WebDriverWait waitPage(){
```

Comment:
de ce ati schimbat modificatorul de acces, din private in public ?

Reply:
Am schimbat in private